deps_licenses/licenses_linux_user.txt

```diff
@@ -4,15 +4,16 @@ MarkupSafe, 2.1.5, BSD License
 PyYAML, 6.0.2, MIT License
 brevitas, 0.10.2, UNKNOWN
 certifi, 2024.8.30, Mozilla Public License 2.0 (MPL 2.0)
-charset-normalizer, 3.3.2, MIT License
+charset-normalizer, 3.4.0, MIT License
 coloredlogs, 15.0.1, MIT License
+concrete-ml-extensions, 0.1.2, BSD-3-Clause-Clear
```

Comment:
only for linux, we'll see in the weekly what happens on mac (should fall back to CP compilation)

Reply:
Did you run make licenses? 

I thought we were only supposed to do that through GitHub Actions. 
Or was it for a another purpose ?

Reply:
yes, I generated the license files in the action. but this library only works on linux so only the linux license file shows it